### ISSUE TITLE
fix the bug about the softmaxlayer

### ISSUE THREAD

**Issue by fengshikun** [ethereon/caffe-tensorflow#70]
When I execute the code below:
```
# Import the converted model's class
from mynet import MyNet

# Create an instance, passing in the input data
net = MyNet({'data':my_input_data})
```
It will cause the tensorflow error:
```
assert all(arg.dtype in dtype_hierarchy for arg in [start, limit, delta])
AssertionError
```

Change the kaffe/tensorflow/network.py line 215:
`return tf.nn.softmax(input, name)`
to
`return tf.nn.softmax(input, name=name)`
fix this bug

**Comment by chiggum** [comment]
tf.nn.softmax(logits=input, name=name)
